source/CargoHold.cpp

```diff
@@ -570,3 +571,27 @@ int CargoHold::IllegalCargoFine() const
 	}
 	return worst;
 }
+
+
+
+// Retrieve vector of pointers to the outfits, sorted descending by size.
+const vector<const Outfit *> CargoHold::GetOutfitOrder() const
+{
+	vector<const Outfit *> sortedOutfits;
+	if(!outfits.size())
+		return sortedOutfits;
+	
+	for(const auto &outfit : outfits)
+		sortedOutfits.emplace_back(outfit.first);
+	sort(sortedOutfits.begin(), sortedOutfits.end(),
+		[] (const Outfit *lhs, const Outfit *rhs)
+		{
+			if(lhs->Get("mass") == rhs->Get("mass"))
+				return lhs->Name() < rhs->Name();
```

Comment:
Note that the comparison function given to sort() only has to be a weak ordering, so this step isn't necessary; if two outfits have the same mass it doesn't matter which one comes first.